src/caf/toolkit/config_base.py

```diff
@@ -220,6 +220,39 @@ def write_example(
 
 
 # # # FUNCTIONS # # #
+def _is_collection(object: any):
```

Comment:
Don't use `object` as a variable name - it's already the name of a built in.

Try `obj` instead?